### ISSUE TITLE
Windows to go is missing in every release

### ISSUE THREAD

**Issue by Randywfl** [pbatard/rufus#2488]
﻿

Checklist
---------
- [ ] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [ ] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [ ] I clicked the 'Log' button (🗒️) or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, or used [DebugView](https://learn.microsoft.com/en-us/sysinternals/downloads/debugview), and copy/pasted the log into the section that says `<FULL LOG>` below.
- [ ] The log I am copying is the FULL log, starting with the line `Rufus version: x.y.z` - I have NOT removed any part of it.

Additionally (if applicable):
- [x] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [x] I also tried one or more of the following:
  - [ ] Using a different USB drive.
  - [x] Plugging the USB into a different port.
  - [x] Running Rufus on a different computer.
- [ ] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------
I have downloaded and tried almost every release from 3.2 up to 4.5 and cannot find a single copy to offer the Windows To Go option.  I thought that was the purpose of Rufus but the option is gone. I've seen the option in various web posts so I know it did exist.  I was trying to create a Windows 7 to go on an SD card plugged into USB port using two different SD card readers using an ISO I created from my official MS DVD Windows 7 Professional that was proven to work on the internal HDD, before the onboard HDD controller went belly up.  I can still boot o n the built-in SD card reader and that is why I am attempting to resurrect this old Asus laptop using SD reader.

Log
---
```
Rufus x64 v4.5.2180 (Portable)
Windows version: Windows 11 Pro x64 (Build 22631.3672)
Syslinux versions: 4.07/2013-07-25, 6.04/pre1
Grub versions: 0.4.6a, 2.12
System locale ID: 0x0409 (en-US)
Will use default UI locale 0x0409
SetLGP: Successfully set NoDriveTypeAutorun policy to 0x0000009E
Localization set to 'en-US'
Notice: The ISO download feature has been deactivated because 'Check for updates' is disabled in your settings.
Found 517 officially revoked UEFI bootloaders from embedded list
Found 2351 additional revoked UEFI bootloaders from this system's SKUSiPolicy.p7b
Found USB device 'SDXC Card' (????:????)
Found UAS (USB 3.0) device 'WDC WD50 00BEVT-00ZAT0 UAS Device' (0BDA:9201)
Device eliminated because it was detected as a Hard Drive (score 3 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
1 device found
Disk type: Removable, Disk size: 256 GB, Sector size: 512 bytes
Cylinders: 31134, Tracks per cylinder: 255, Sectors per track: 63
Partition type: MBR, NB Partitions: 1
Disk ID: 0x086E7B4A
Drive has a Rufus Master Boot Record
Partition 1:
  Type: NTFS (0x07)
  Detected File System: NTFS
  Size: 238.5 GB (256086376448 bytes)
  Start Sector: 2048, Boot: Yes
Scanning image...
ISO analysis:
  Image is a UDF image
Disk image analysis:
  Image does not have a Boot Marker
ISO label: 'GRMCPRFRER_EN_DVD'
  Detected: Windows 7 ISO (Build 7600.16385)
  Size: 2.3 GB (Projected)
  Uses: Bootmgr (BIOS only)
  Uses: Install.wim (version 0.13.1)
Using image: Windows7.ISO (2.3 GB)
Found USB device 'SDXC Card' (????:????)
Found UAS (USB 3.0) device 'WDC WD50 00BEVT-00ZAT0 UAS Device' (0BDA:9201)
Device eliminated because it was detected as a Hard Drive (score 3 > 0)
If this device is not a Hard Drive, please e-mail the author of this application
NOTE: You can enable the listing of Hard Drives under 'advanced drive properties'
1 device found
Disk type: Removable, Disk size: 256 GB, Sector size: 512 bytes
Cylinders: 31134, Tracks per cylinder: 255, Sectors per track: 63
Partition type: MBR, NB Partitions: 1
Disk ID: 0x086E7B4A
Drive has a Rufus Master Boot Record
Partition 1:
  Type: NTFS (0x07)
  Detected File System: NTFS
  Size: 238.5 GB (256086376448 bytes)
  Start Sector: 2048, Boot: Yes
```

**Comment by Randywfl** [comment]
I suppose I should add that I created my ISO using Imgburn, but I don't see how that should affect Rufus showing the Windows to go option.  And if I let Rufus burn the SD card without the windows to go option, It successfully creates a bootable Windows 7 professional installation USB which does me no goof at all, even though it runs perfectly.

**Comment by thecatontheceiling** [comment]
why did you make a duplicate of the issue you made earlier?

**Comment by pbatard** [comment]
> `Detected: Windows 7 ISO (Build 7600.16385)`

https://github.com/pbatard/rufus/wiki/FAQ#user-content-Windows_XP_Windows_Vista_Windows_7_NO_LONGER_SUPPORTED

**Comment by Randywfl** [comment]
Hi Pete,
 
How very rude of me (tail between my legs).  I can see you stay very busy and I 
did not even think to thank you for taking time to respond to my post on 
GitHub.  
You've done a lot and been very generous with a lot of really great work  Thank you very much for all you have done.  Unfortunately for me I didn't 
get in early enough to take advantage of it.
 
 
Thanks,

Randy Walker





----- Original Message -----
From: Pete Batard ***@***.***>
Reply-To: pbatard/rufus 
***@***.***>
To: pbatard/rufus ***@***.***>
Cc: Randywfl ***@***.***>, Author ***@***.***>
Sent: 6/6/2024 5:24:05 AM
Subject: Re: [pbatard/rufus] Windows to go is missing in every release (Issue 
#2488)



Closed #2488 as completed.
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Comment by Randywfl** [comment]
Hi Lyssa,
 
Sorry about that. GitHab noobie.  Needed to edit [xx] to be just [c] so my post 
would not be ignored, and did not see any option to edit my post so I clicked 
my back button made the edit and clicked send again.  So sorry.
 
 
Thanks,

Randy Walker





----- Original Message -----
From: Lyssa ***@***.***>
Reply-To: pbatard/rufus 
***@***.***>
To: pbatard/rufus ***@***.***>
Cc: Randywfl ***@***.***>, Author ***@***.***>
Sent: 6/6/2024 3:52:06 AM
Subject: Re: [pbatard/rufus] Windows to go is missing in every release (Issue 
#2488)



why did you make a duplicate of the issue you made earlier?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: 
***@***.***>
